### ISSUE TITLE
Two remaining normative references

### ISSUE THREAD

**Issue by llemeurfr** [w3c/lpf#13]
There are 2 remaining normative references, even if all sections now have an "informative" class. 
They should logically come from the new Conformance section.

**Comment by iherman** [comment]
Indeed... I tried setting the 'noRecTrack' to true (which should be done in any case) but respec does that. Let us leave as is, and if the Webmaster complains, I can change that manually before publication.
